src/main/java/com/google/code/gwt/crop/client/GWTCropperPreview.java

```diff
@@ -94,6 +94,8 @@ public GWTCropperPreview(Dimension dimension, int value){
      * @param aspectRatio
      */
     public void init(String imageUrl, int canvasWidth, int canvasHeight, double aspectRatio) {
+        if (embeddedImage != null) remove(embeddedImage);

```

Comment:
Hey, just a quick question: is it possible the situation when an image for preview has been initialized before the Preview Widget itself?